### ISSUE TITLE
Spawning terminal in container breaks with space in path to kubectl

### ISSUE THREAD

**Issue by jplitza** [vscode-kubernetes-tools/vscode-kubernetes-tools#1245]
I installed kubectl system-wide in "C:\Program Files\Git\usr\local\bin\kubectl.exe", but the space caused an error when launching a terminal in a pod from the extension pane.

Apparently there are too many quotes added. In the Host Pty log the following line looks suspicious:
```
2024-01-16 14:43:08.824 [trace] [RPC Request] PtyService#createProcess({"name":"bash on pod/container","executable":"\"C:\\Program Files\\Git\\usr\\local\\bin\\kubectl.exe.exe\"","args":["exec","-it",...
```

**Comment by tejhan** [comment]
Hi, thanks for opening this.

I know it's a bit late, but were you able to specifically confirm it was the spacing causing the issue?

I ask because I notice `kubectl.exe.exe` which leads me to believe there was a string parsing issue, we have logic that appends .exe to a path if it isn't already there, but in this case there appears to be a bug. Is your kubectl path manually defined?

**Comment by jplitza** [comment]
Yes, it was manually defined. And yes, that .exe.exe indeed looks suspicous, too! However, I don't have that setup anymore, so I cannot test.
